config/config.go

```diff
@@ -69,6 +69,7 @@ var (
 		MetricsPath: "/metrics",
 		Scheme:      "http",
 		HonorLabels: false,
+		Proxy: URL{},
```

Comment:
No need to default zero values unless it's worth to explicitly state it like in `HonorLabels`.
But not seeing a proxy in this struct intuitively means that there is none.